onnxruntime/contrib_ops/cpu/transformers/beam_search_impl_base.h

```diff
@@ -141,7 +141,7 @@ class BeamSearchBase : public GenerateBase  {
     parameters_->ParseFromInputs(&context);
   }
 
-  virtual ~BeamSearchBase(){}
+  virtual ~BeamSearchBase() {}
```

Comment:
"virtual" is for virtual functions in the base class. Here you should use override.